src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -523,3 +523,67 @@
     choices=utils.get_enum_values(TriggerState),
     help="List all the schedules in the resource which are in the provided states.",
 )
+
+notification_rule_test_ids = CLIArgumentType(
+    options_list=["--test-ids"],
+    nargs="+",
+    validator=validators.validate_notification_rule_test_ids,
+    help="Test ids for notification rule.",
+)
+
+notification_display_name = CLIArgumentType(
+    option_list=["--display-name"],
+    type=str,
+    help="Display name of notification rule.",
+)
+
+notification_rule_id = CLIArgumentType(
+    validator=validators.validate_notification_rule_id,
+    option_list=["--notification-rule-id"],
+    type=str,
+    help="Identifier for the notification rule.",
+)
+
+action_groups = CLIArgumentType(
+    option_list=["--action-groups"],
+    nargs="*",
+    help="Action group for notification rule.",
```

Comment:
Space separated list of resource ids of action groups for the notification rule.